### ISSUE TITLE
`%chatchat_channel_prefix%` is incorrect when using message prefixes

### ISSUE THREAD

**Issue by lucypoulton** [HelpChat/ChatChat#27]
As the user's channel is never actually updated, the placeholder results in the user's current active channel. This poses issues when using that placeholder in messages to denote their channel

**Comment by BlitzOffline** [comment]
Yeah we're aware. We're currently looking int good solutions for this. We could use some recommendations. At first we just thought we should temp update the channel but that might come with some problems as well
